python/lsst/ts/wep/task/cutOutDonutsBase.py

```diff
@@ -727,10 +727,6 @@ def cutOutStamps(self, exposure, donutCatalog, defocalType, cameraName):
             # Calculate the S/N per stamp
             snQuant.append(self.calculateSN(donutStamp))
 
-            # Add back in measured mean background to get mean background
```

Comment:
And that's not needed since we actually never removed background in `cutOutDonutsBase`, right? (I know we do +/-  background in `generateDonutsDirectDetect`  )

Reply:
We actually run the background task at the start of `cutOutStamps` which does subtract the background from the exposure that we use to cut out the donuts.